### ISSUE TITLE
[Feature Request] Buttons ( to trigger scripts) inside cardMedia

### ISSUE THREAD

**Issue by finipini** [joBr99/nspanel-lovelace-ui#255]
Hello. I would like to have 1 or more buttons inside cardMedia to active some scripts that would trigger some playlists in spotify or some radios in alexa,s.  The reason is that the cardMedia is only usefull for me if spotify is already playing or in the case of alexa , only to control volume and source.  Would be great if i also could choose what to play.

I atched a photo from an mini-media-player card that i have with 3 buttons who will trigger automations to start diferent radios just to demonstrate an example of my request.

**Additional context**

**Panel / Firmware Version**
I, m using the EU version with v2.9.0.
![Screenshot_47](https://user-images.githubusercontent.com/49859284/171285032-c0810625-77b6-461b-a8ec-6507009cbc8b.jpg)

**Comment by finipini** [comment]
Yes  is not a lot space there.... and it can look a little messy but somewhere vertically between the title and the pause button, one or in maximum 2 buttons....

**Comment by joBr99** [comment]
Title and Interpret can be really long, the only spot where currently no ui element exists is left to the previous button

**Comment by finipini** [comment]
![photo_2022-05-31_22-58-28](https://user-images.githubusercontent.com/49859284/171291323-96bb982e-e13e-457d-add3-b5448cacc66f.jpg)

Yes not a lot o space here , never notested where interpret and title could go......

One iis better than nothing, but i see your point....  my first thought was really somethig just to turn on the media players, if just one button maybe have to play with scripts and do a random selection or make some rules depending the time of the day.

The icon of musical note is there to match with the song title?  

Is there something in the right spot of the "next" button ?

**Comment by joBr99** [comment]
Yes, if your media player supports on/off, there is a power button.

**Comment by joBr99** [comment]
I will try add a button that can be configured like on cardGrid, so you can use it to navigate to a hidden card or run scripts.

**Comment by joBr99** [comment]
works for me, you can configure a navigate entity there or a script (or a light or a switch)

note that the icon doesn't change based on the configured mediaControl value, it is still showing the mediaType of the currently played stuff

```
      - type: cardMedia
        entity: media_player.spotify
        mediaControl: navigate.type_key
```

**Comment by erugaman** [comment]
Thanks! It´s a great addition!!
